cmd/flow-dps-indexer/main.go

```diff
@@ -109,8 +115,8 @@ func run() int {
 	log = log.Level(level)
 
 	// Ensure that at least one index is specified.
-	if !flagIndexAll && !flagIndexCommit && !flagIndexHeader && !flagIndexPayloads &&
-		!flagIndexCollections && !flagIndexTransactions && !flagIndexEvents {
+	if !flagIndexAll && !flagIndexCommit && !flagIndexHeader && !flagIndexPayloads && !flagIndexCollections &&
+		!flagIndexTransactions && !flagIndexResults && !flagIndexEvents && !flagIndexSeals {
```

Comment:
Is there any value in separating transaction and result indexing?

Reply:
I didn't think so either but you said it should be the case [in an earlier review](https://github.com/optakt/flow-dps/pull/260#discussion_r665400565) 🤷 I'm fine with reverting it again if you prefer.

Reply:
Let's leave it like this. I think the most important part is being able to incrementally upgrade an index. So that would fit the bill.